src/backend/columnar/columnar_tableam.c

```diff
@@ -2363,6 +2364,55 @@ ColumnarProcessUtility(PlannedStmt *pstmt,
 				(errmsg("columnar storage parameters specified on non-columnar table")));
 	}
 
+	if (IsA(parsetree, CreateExtensionStmt))
+	{
+		CreateExtensionStmt *createExtensionStmt = castNode(CreateExtensionStmt,
+															parsetree);
+
+		if (get_extension_oid("citus_columnar", true) == InvalidOid)
+		{
+			if (strcmp(createExtensionStmt->extname, "citus_columnar") == 0)
+			{
+				DefElem *newVersionValue = GetExtensionOption(
+					createExtensionStmt->options,
+					"new_version");
+
+				/*we are not allowed to install citus_columnar as version 11.1-0 by cx*/
+				if (newVersionValue)
+				{
+					const char *newVersion = defGetString(newVersionValue);
+					if (strcmp(newVersion, "11.1-0") == 0)
```

Comment:
maybe use a constant for "11.1-0" whose name describes its means

Reply:
done